server/src/main/java/com/project/cisco/database/entity/Language.java

```diff
@@ -0,0 +1,23 @@
+package com.project.cisco.database.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.FieldNameConstants;
+
+@Data
+@ToString(callSuper = true)
+@FieldNameConstants
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+public class Language {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
+
+    @Column
```

Comment:
Add  `@NotBlank` here, please

it will require `import javax.validation.constraints.NotBlank;`
so you need to add this dependency:

```
<dependency> 
    <groupId>org.springframework.boot</groupId> 
    <artifactId>spring-boot-starter-validation</artifactId> 
</dependency>
```

Reply:
I would also put a constrain on a size, eg.:
`@Size(min = 2, max = 32, message = "Language name must be between 2 and 32 characters long") `

Reply:
Done